File: 1-Leccion-Operaciones-Basicas/D.Repartiendo-cachorros.cpp
#include <iostream>
using namespace std;

int main() {
  int C, P, H;
  cin >> C >> P >> H;

  int seQueda = P;
  C -= P;

  seQueda += C % (H + 1);

  cout << seQueda << endl;

  return 0;
}

File: 1-Leccion-Operaciones-Basicas/D.Repartiendo-cachorros.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);    
   
    int C = scanner.nextInt();
    int P = scanner.nextInt();
    int H = scanner.nextInt();

    int seQueda = P;
    C -= P;

    seQueda += C % (H + 1);

    System.out.println(seQueda);
  }
}